The usual gcc-3.4 fixes.

  - Rene Rebe <rene@exactcode.de>
  - Andreas V. Meier <avmeier@web.de>

diff -ruN tree-1.4/tree.c tree-1.4-new/tree.c
--- tree-1.4/tree.c	2003-02-06 17:24:11.000000000 +0100
+++ tree-1.4-new/tree.c	2004-05-15 22:09:13.994582920 +0200
@@ -1083,6 +1083,7 @@
       case '?':
       case '+':
       default:
+      ((void)0);
     }
   }
 }
